third_party/openthread/CMakeLists.txt

```diff
@@ -114,3 +114,9 @@ if (NOT OT_THREAD_VERSION STREQUAL "1.1")
         "-DOPENTHREAD_CONFIG_TMF_PROXY_MLR_ENABLE=1"
     )
 endif()
+
+if (OTBR_EPSKC)
+    target_compile_definitions(ot-config INTERFACE "-DOPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE=1" )
+else()
+    target_compile_definitions(ot-config INTERFACE "-DOPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE=0" )
+endif()
```

Comment:
How about we force enabling it just like other features?

Reply:
How about we have the option here and set OTBR_EPSKC to ON by default? Then we have a method to turn on and off. What's more, keep consistency for OTBR and OT ePSKc feature enable/disable at the same time.